manifests/c/cuiliang/quicker/1.22.43/cuiliang.quicker.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: cuiliang.quicker
+PackageVersion: 1.22.43

```

Again the version number
```suggestion
PackageVersion: 1.22.43.0
```